lgc/test/StackBoundsCheck.lgc

```diff
@@ -0,0 +1,128 @@
+; RUN: lgc -mcpu=gfx900 -stop-after=lgc-bounds-check-memory - <%s | FileCheck %s
+
+; lgc code generated with amdllpc llpc/test/shaderdb/OpLoad_StackBoundsCheck_struct.pipe -emit-lgc
+
+; CHECK:      = fmul reassoc nnan nsz arcp contract afn float %[[#]], 2.550000e+02
+; CHECK-NEXT: = fptosi float %[[#]] to i32
+; CHECK-NEXT: = icmp ult i32 %[[#]], -16
+; CHECK-NEXT: = xor i1 %[[#]], true
+; CHECK-NEXT: = call { i1, i64 } @llvm.amdgcn.if.i64(i1 %.inv)
+; CHECK-NEXT: = extractvalue { i1, i64 } %[[#]], 0
+; CHECK-NEXT: = extractvalue { i1, i64 } %[[#]], 1
```

Comment:
Can you reference concrete instruction names to make sure the operands are like you'd expect?